scripts/setup-seafile-database.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
I disagree with your point of view.  The wrapper script only checks for the required pre-reqs like python versions and essential python plugins for the databases.  It is better to have one script checking for that common functionality rather than duplicate into two.  Also come upgrade time it will be easier to have this format.